src/layer/loongarch/sigmoid_loongarch.cpp

```diff
@@ -0,0 +1,76 @@
+// Leo is pleased to support the open source community by making ncnn available.
+//
+// Copyright (C) 2020 Leo <leo@nullptr.com.cn>. All rights reserved.
```

Comment:
fix copyright header
use year 2022
either your author info or tencent is fine

Reply:
fix